consensus/vbft/service.go

```diff
@@ -1190,8 +1194,9 @@ func (self *Server) processProposalMsg(msg *blockProposalMsg) {
 				log.Errorf("server %d verify proposal blk from %d timedout, blk %d, txs %d, err: %s",
 					self.Index, msg.Block.getProposer(), msgBlkNum, len(txs), err)
 			}
+			nonceCtx := make(map[common.Address]uint64)
 			for _, tx := range txs {
-				if err := self.incrValidator.Verify(tx, validHeight); err != nil {
+				if err := self.incrValidator.Verify(tx, validHeight, nonceCtx); err != nil {
```

Comment:
nonceCtx is not used after Verify and we create it a lot,  can we put this to incrValidator?